src/main/java/com/cloudbees/jenkins/support/impl/BuildQueue.java

```diff
@@ -69,21 +66,20 @@ public String getDisplayName() {
 
   @Override
   public void addContents(@NonNull Container container) {
-    container.add(new Content("buildqueue.md") {
+    container.add(new PrefilteredPrintedContent("buildqueue.md") {
```

Comment:
Only the private information is filtered to avoid destroying the markdown structure